### PR TITLE
Do not use dune 3.0

### PR DESCRIPTION
**Pull request by gpetiot**
See https://github.com/ocaml/opam-repository/pull/20765

### CODE REVIEW

File: ocamlformat.opam [hunk 1]
```diff
@@ -8,11 +8,11 @@ authors: ["Josh Berdine <jjb@fb.com>"]
 homepage: "https://github.com/ocaml-ppx/ocamlformat"
 bug-reports: "https://github.com/ocaml-ppx/ocamlformat/issues"
 depends: [
-  "dune" {>= "2.8"}
   "ocaml" {>= "4.08" & < "4.15"}
   "alcotest" {with-test}
   "base" {>= "v0.12.0" & < "v0.15"}
   "cmdliner" {>= "1.1.0"}
+  "dune" {>= "2.8" & with-test & < "3.0"}
```
Comment (Julow) [inline ocamlformat.opam:15]: I think you meant this:
```suggestion
  "dune" {>= "2.8"}
  "dune" {with-test & < "3.0"}
```

**Review by Julow — APPROVED** [review]

File: ocamlformat.opam (hunk 1 above)
Comment (gpetiot) [reply ocamlformat.opam:15]: Yes but this is generated from the `dune-project` file

Comment (Julow) [reply ocamlformat.opam:15]: Oups! You are right. I've found a way to fix this: https://github.com/gpetiot/ocamlformat/pull/10

Comment (gpetiot) [reply ocamlformat.opam]: Thanks!

**Pull request merged**